code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1114,3 +1114,83 @@
 		if(initial(liber.filterToxins))
 			liber.filterToxins = TRUE
 	L.metabolism_efficiency += metab_inibition
+
+/datum/reagent/toxin/radscorp
+	name = "Radscorpion Venom"
+	description = "Venom extracted from a radscorpion."
+	color = "#8C7918" // rgb: 140, 121, 24
+	taste_description = "sweetness"
+	taste_mult = 1.2
+	value = REAGENT_VALUE_COMMON //Encouraging people to mix toxins for reasons beyond harming each other or mixing reagents such as pen acid.
+	toxpwr = 1.3
+	ghoulfriendly = TRUE
+
+/datum/reagent/toxin/cazador
```

Comment:
theres actually cazador venom in the codebase called cazador_venom, wouldn't using that be better?

Reply:
Changed to the existing on and removed the duplicate I added.